.circleci/config.yml

```diff
@@ -36,31 +40,28 @@ jobs:
       - run:
           name: "Run build"
           command: 'yarn run build'
+      - persist_to_workspace:
```

Comment:
this stores everything in the repo folder in the workspace volume, such that other jobs in the workflow can reuse the node_modules + build artifacts in dist/. This way they don't have to rebuild the whole thing themselves